rcon/player_history.py

```diff
@@ -227,6 +231,31 @@ def ban_if_blacklisted(rcon, steam_id_64, name):
             )
 
 
+def ban_if_has_vac_bans(rcon, steam_id_64, name):
+    try:
+        autoban_threshold = int(AUTOBAN_TRIGGER_THRESHOLD)
+    except ValueError: # No (proper) value is given
+        return
+
+    with enter_session() as sess:
+        player = get_player(sess, steam_id_64)
+
+        if not player:
+            logger.error("Can't check VAC history, player not found %s", steam_id_64)
+            return
+
+        bans = get_player_bans(steam_id_64)
+        if bans == {}: return # Player has no bans
+        last_ban = bans['DaysSinceLastBan']
+        if int(last_ban) <= autoban_threshold:
```

Comment:
Are you sure this is castable to an int? I thought this could be null?

I'd rename those a bit to be clearer
```
int(last_ban) <= autoban_threshold:
```

to something of the sort:

```
days_since_last_ban <= max_days_since_ban
```